code/game/objects/structures/grille.dm

```diff
@@ -302,3 +302,40 @@
 	rods_broken = FALSE
 	grille_type = /obj/structure/grille
 	broken_type = null
+
+/obj/structure/grille/prison //grilles that trigger prison lockdown under some circumstances
+	name = "prison grille"
+	desc = "a set of rods under current used to protect the prison wing. An alarm will go off if they are breached."
+	var/obj/item/assembly/control/device
```

Comment:
Make sure to delete this device inside /Destroy(), since it isn't a part of the objects contents

Reply:
Adressed in destroy()